### PR TITLE
add cpu.idle support

### PR DESCRIPTION
**Pull request by wineway**
Signed-off-by: wineway <wangyuweihx@gmail.com>

### CODE REVIEW

**Comment by wineway** [comment]
@utam0k PTAL

File: src/runtime/linux.rs [hunk 1]
```diff
@@ -345,6 +345,11 @@ pub struct LinuxCpu {
     /// CPU hardcap limit (in usecs). Allowed cpu time in a given period.
     quota: Option<i64>,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[getset(get_copy = "pub", set = "pub")]
+    /// cgroups are configured with minimum weight, 0: default behavior, 1: SCHED_IDLE.
```
Comment (utam0k) [inline src/runtime/linux.rs:350]:
```suggestion
    /// Cgroups are configured with minimum weight, 0: default behavior, 1: SCHED_IDLE.
```

File: src/runtime/linux.rs [hunk 2]
```diff
@@ -345,6 +345,11 @@ pub struct LinuxCpu {
...
+    idle: Option<i64>,
```
Comment (utam0k) [inline src/runtime/linux.rs:351]: 👍 
https://github.com/opencontainers/runtime-spec/blob/27924127bf391ea7691924c6dcb01f3369d69fe2/schema/config-linux.json#L122-L124

**Review by utam0k — COMMENTED** [review]
Thanks for your request. Can I ask you to fix a nit?

**Comment by wineway** [comment]
> Thanks for your request. Can I ask you to fix a nit?

@utam0k sure😂, fixed

**Review by utam0k — APPROVED** [review]
💯

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**